Database/Updates/World/04-06-20-2017-alterPostionTable-stored.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `ace_world`.`ace_position`
+  DROP COLUMN `landblock` ,
+  ADD COLUMN `landblock` INT(5) UNSIGNED AS ( ((landblockraw >> 16) & 0xFFFF)) STORED,
```

Comment:
I ran the test again - it is faster stored on query (not insert) but it is not that impactful.   I am ok with changing this back if it makes sense.